_posts/0000-01-05-evergreene.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Spotted Puss
```

Comment:
No comment for fear of being had up in front of HR

Reply:
Done